src/components/DataDictionary/components/Entity/entity.styles.ts

```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+import { Typography } from "@mui/material";
+import { LayoutSpacing } from "../../hooks/UseLayoutSpacing/types";
+import { ENTITIES_ROW_GAP } from "../Entities/constants";
+
+export const StyledTypography = styled(Typography)<Partial<LayoutSpacing>>`
+  scroll-margin-top: ${({ top = 0 }) => top + ENTITIES_ROW_GAP}px;
```

Comment:
It is expected that the scroll-margin-top value will be adjusted when filtering toolbar is completed.